Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -178,52 +178,52 @@ CEXIMemoryCard::GetGCIFolderPath(int card_index, AllowMovieFolder allow_movie_fo
 void CEXIMemoryCard::SetupGciFolder(const Memcard::HeaderData& header_data)
 {
   const std::string& game_id = SConfig::GetInstance().GetGameID();
-  u32 CurrentGameId = 0;
+  u32 current_game_id = 0;
   if (game_id.length() >= 4 && game_id != "00000000" &&
       SConfig::GetInstance().GetTitleID() != Titles::SYSTEM_MENU)
   {
-    CurrentGameId = Common::swap32(reinterpret_cast<const u8*>(game_id.c_str()));
+    current_game_id = Common::swap32(reinterpret_cast<const u8*>(game_id.c_str()));
   }
 
   // TODO(C++20): Use structured bindings when we can use C++20 and refer to structured bindings
   // in lambda captures
   const auto folder_path = GetGCIFolderPath(card_index, AllowMovieFolder::Yes);
-  const auto& strDirectoryName = folder_path.first;
+  const auto& dir_name = folder_path.first;
```

Comment:
This is actually a path, so I would prefer `dir_path`. (Maybe also `std::string` instead of `auto`, just to make things as clear as possible?)

Reply:
Addressed